src/drive/store/configureStore.js

```diff
@@ -35,7 +36,7 @@ const configureStore = options => {
     middlewares.push(createTrackerMiddleware())
   }
 
-  if (options.logger !== false) {
+  if (flag('drive.logger') === true) {
```

Comment:
Isn't the equality comparison to `true` redundant? If it's a truthy value and we only want strict true, maybe we can convert the result with `Boolean()` or double NOT operator `!!`. This is just a style concern. Though from looking at the flag it would be strange to me to have truthy values that I don't want (but then it's a cozy-flags issue)

Reply:
We can have flags that are not boolean, but it will be in this case, so `if (flag('drive.logger')` should be enough